tests/get_search.test.ts

```diff
@@ -96,7 +96,8 @@ describe.each([
 
     const response = await client.index(index.uid).searchGet('prince', {});
 
-    expect(response).toHaveProperty('hits', expect.any(Array));
+    expect(response).toHaveProperty('hits');
+    expect(Array.isArray(response.hits)).toBe(true);
```

Comment:
Old way was checking with `instanceof` internally, but it is clearly sated in the docs that arrays are not to be checked this way: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#description